clients/gtest/rocblas_gtest_main.cpp

```diff
@@ -12,6 +12,8 @@
 
 int main(int argc, char** argv)
 {
+    // Open data file
```

Comment:
In the rocblas_gtest_main main() program, we initialize the RocBLAS_TestData class instantiation with the data file name.